packages/plugin-core/src/showcase/GraphPanelTip.ts

```diff
@@ -0,0 +1,32 @@
+import { ShowcaseEntry } from "@dendronhq/engine-server";
+import * as vscode from "vscode";
+import {
+  DisplayLocation,
+  IFeatureShowcaseMessage,
+} from "./IFeatureShowcaseMessage";
+import { DendronTreeViewKey } from "@dendronhq/common-all";
+
+export class GraphPanelTip implements IFeatureShowcaseMessage {
+  shouldShow(_displayLocation: DisplayLocation): boolean {
+    return true;
+  }
+  get showcaseEntry(): ShowcaseEntry {
+    return ShowcaseEntry.GraphPanel;
+  }
+
+  getDisplayMessage(_displayLocation: DisplayLocation): string {
+    return "Dendron has a new Graph Panel. Check it out!";
```

Comment:
nit: Can we amend this message to maybe also help with side bar discoverability? 

"We're experimenting with a note graph panel in the new Dendron sidebar.  Check it out!"